plugins/UM3NetworkPrinting/src/SendMaterialJob.py

```diff
@@ -1,99 +1,129 @@
 # Copyright (c) 2018 Ultimaker B.V.
 # Cura is released under the terms of the LGPLv3 or higher.
 
-import json #To understand the list of materials from the printer reply.
-import os #To walk over material files.
-import os.path #To filter on material files.
-from PyQt5.QtNetwork import QNetworkReply, QNetworkRequest #To listen to the reply from the printer.
-from typing import Any, Dict, Set, TYPE_CHECKING
-import urllib.parse #For getting material IDs from their file names.
-
-from UM.Job import Job #The interface we're implementing.
+import json  # To understand the list of materials from the printer reply.
+import os  # To walk over material files.
+import os.path  # To filter on material files.
+import urllib.parse  # For getting material IDs from their file names.
+from typing import Dict, TYPE_CHECKING
+
+from PyQt5.QtNetwork import QNetworkReply, QNetworkRequest  # To listen to the reply from the printer.
+
+from UM.Job import Job  # The interface we're implementing.
 from UM.Logger import Logger
-from UM.MimeTypeDatabase import MimeTypeDatabase #To strip the extensions of the material profile files.
+from UM.MimeTypeDatabase import MimeTypeDatabase  # To strip the extensions of the material profile files.
 from UM.Resources import Resources
-from UM.Settings.ContainerRegistry import ContainerRegistry #To find the GUIDs of materials.
-
-from cura.CuraApplication import CuraApplication #For the resource types.
+from UM.Settings.ContainerRegistry import ContainerRegistry  # To find the GUIDs of materials.
+from cura.CuraApplication import CuraApplication  # For the resource types.
+from plugins.UM3NetworkPrinting.src.Models import ClusterMaterial, LocalMaterial
 
 if TYPE_CHECKING:
     from .ClusterUM3OutputDevice import ClusterUM3OutputDevice
 
+
 ##  Asynchronous job to send material profiles to the printer.
 #
 #   This way it won't freeze up the interface while sending those materials.
 class SendMaterialJob(Job):
     def __init__(self, device: "ClusterUM3OutputDevice") -> None:
         super().__init__()
-        self.device = device #type: ClusterUM3OutputDevice
+        self.device = device  # type: ClusterUM3OutputDevice
 
     def run(self) -> None:
-        self.device.get("materials/", on_finished = self.sendMissingMaterials)
+        self.device.get("materials/", on_finished=self.sendMissingMaterials)
```

Comment:
Cura uses spaces around assignments.

Reply:
That's not really an assignment though. It's just parameter passing.